languages/clojure/exercises/concept/numbers/.meta/example.clj

```diff
@@ -0,0 +1,21 @@
+(ns numbers)
+
+(def default-rate 221)
+
+(defn success-rate [speed]
+  (cond (= speed 10) 0.77
+        (= speed 9) 0.8
```

Comment:
@porkostomus hah I was thinking what happens when speed is between 9 and 10...and found out the table at the beginning didn't mention that, so the speed here is supposed to be integers only, that should be fine